README.md

```diff
@@ -6,10 +6,12 @@ This repository contains examples showing how to consume the PowerDMS File Picke
 
 The File Picker is implemented as an IFrame, with the contents hosted at `https://filepicker.powerdms.com`.
 
-There is a small initialization script and stylesheet that must be referenced on the consuming page to create and cleanup the IFrame. The initialization script will add a function called `initializePowerDmsFilePicker` to the `window` which will create the IFrame. The consuming page must have a button that will call this function.
+There is a small [initialization script and stylesheet](https://github.com/PowerDMS/PowerDMS/tree/develop/src/clients/file-picker/initializer) that must be referenced on the consuming page to create and cleanup the IFrame. The initialization script will add a function called `initializePowerDmsFilePicker` to the `window` which will create the IFrame. The consuming page must have a button that will call this function.

```

Comment:
🔧 We can't make links to the `PowerDMS` repository because that is private. Also, users should not be concerned with the implementations of these files. By importing the URLs specified in the `How to integrate` solution they will get what they need. They can also visit those URLs directly if they are really curious what these files look like.